Client/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -2285,6 +2285,27 @@ bool CStaticFunctionDefinitions::SetPedAnimationProgress(CClientEntity& Entity,
     return false;
 }
 
+float CStaticFunctionDefinitions::GetPedAnimationProgress(CClientEntity& entity)
+{
+    CClientPed& ped = static_cast<CClientPed&>(entity);
+
+    auto* currentTask = ped.GetTaskManager()->GetActiveTask();
+    auto  type = currentTask->GetTaskType();
```

Comment:
Please, do not use auto instead of short or implicit types. The same for the below.